cloudinary/uploader.py

```diff
@@ -356,13 +419,13 @@ def call_api(action, params, http_headers=None, return_error=False, unsigned=Fal
         else:
             params = utils.sign_request(params, options)
 
-        param_list = OrderedDict()
+        param_list = []
```

Comment:
This refactoring can be bit dangerous potentially, since all uploader methods use this function and side effects can arise in a very unexpected places. Let's keep it and see how it goes, this seems to be the right way of sending list parameters.